pipelines/internal/commands/run/run.go

```diff
@@ -213,6 +216,13 @@ func runPipeline(ctx context.Context, service *genomics.Service, req *genomics.R
 	abort := make(chan os.Signal, 1)
 	signal.Notify(abort, os.Interrupt)
 
+	topic, err := newPubSubTopic(ctx, req.Pipeline.Resources.ProjectId)
```

Comment:
I don't think we want to build topics in the tooling - gcloud can be used for that - I think we just want to support an optional flag.

Reply:
I think this is one feature that should be enabled by default. I changed the functionality to always use projects/<current-project>/topics/pipelines-tool if only it has the lable created-by:pipelines-tool. If the tool has any problem in using Pub/Sub it will silently switch to long pooling. Also the user can specify --pub-sub=false to opt for long pooling.